### PR TITLE
Update polish translation of theme/base/account properties

### PR DESCRIPTION
**Pull request by li-on**
Updated polish translation for thehe/base/account properties file

### CODE REVIEW

File: themes/src/main/resources-community/theme/base/account/messages/messages_pl.properties [hunk 1]
```diff
@@ -246,4 +267,140 @@ havePermissionRequestsWaitingForApproval=Masz {0} żądań uprawnień oczekując
...
+filterByName=Filtruj wg nazwy ...
+allApps=Wszystkie aplikacje
+internalApps=Wewnętrzne aplikacje
+thirdpartyApps=Zewnętrzne aplikacje
+appResults=Wyniki
+clientNotFoundMessage=Nie znaleziono klienta
+
+# Linked account
+authorizedProvider=Dostawca logowania
+authorizedProviderMessage=Dostawca logowania  powiązany z kontem
```
Comment (agagancarczyk) [inline themes/src/main/resources-community/theme/base/account/messages/messages_pl.properties:309]: There are two spaces between "logowania"  but only one is required "powiązany"

**Review by agagancarczyk — CHANGES_REQUESTED** [review]

**Comment by agagancarczyk** [comment]
Thanks @li-on I left a small comment for you to address. Also please do a sign-off on the commit using `git -s -m "commit-message"` so that DCO does not fail.

File: themes/src/main/resources-community/theme/base/account/messages/messages_pl.properties (hunk 1 above)
Comment (li-on) [reply themes/src/main/resources-community/theme/base/account/messages/messages_pl.properties]: Corrected

**Review by agagancarczyk — DISMISSED** [review]
LGTM!

**Review by ahus1 — CHANGES_REQUESTED** [review]
Thank you for this update of the translations. There is one small thing to change below, please have a look. 

I know this should be be better documented, and we should probably have a test for this as well :-( but that is out of scope for this PR.

File: themes/src/main/resources-community/theme/base/account/messages/messages_pl.properties [hunk 2]
```diff
@@ -246,4 +267,140 @@ havePermissionRequestsWaitingForApproval=Masz {0} żądań uprawnień oczekując
...
+locale_no=Norsk
+locale_pl=Polski
+locale_pt-BR=Português (Brasil)
+locale_ru=Русский
+locale_sk=Slovenčina
+locale_sv=Svenska
+locale_th=ไทย
+locale_tr=Türkçe
+locale_zh-CN=中文简体
+locale_fi=Suomi
```
[the remark this answers is not in the document]
Comment (li-on) [reply themes/src/main/resources-community/theme/base/account/messages/messages_pl.properties]: Removed

**Review by ahus1 — APPROVED** [review]
Thank you for this change!

**Pull request merged**